Spigot-API-Patches/0185-Ticket-level-per-player-view-distance-API.patch

```diff
@@ -0,0 +1,56 @@
+From 5892e7348815777aceeac44572836ff9013df7d7 Mon Sep 17 00:00:00 2001
+From: slicklibro <slicklibro@gmail.com>
+Date: Sat, 28 Sep 2019 00:01:23 +1200
+Subject: [PATCH] Ticket-level per player view distance API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 8d8d8957..d4415984 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1861,6 +1861,9 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+ 
+     /**
+      * Gets the view distance for this player
++     * <br>
++     * This determines how far the player can add tickets to chunks.
++     * This is bound under the player's tracking view distance
+      *
+      * @return the player's view distance
+      */
+@@ -1868,10 +1871,31 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+ 
+     /**
+      * Sets the view distance for this player
+-     *
++     * <br>
++     * This determines how far the player can add tickets to chunks.
++     * This is bound under the player's tracking view distance
++     * 
+      * @param viewDistance the player's view distance
+      */
+     public void setViewDistance(int viewDistance);
++
++    /**
++     * Gets the tracking view distance for this player
```

Comment:
why do we need a tracking distance separately in api

Reply:
for shared view distance. this is a massive benefit that cannot be lost.